File: pyjl/tests/tests_new/rounding_test.py
from decimal import *

if __name__ == "__main__":
    getcontext().prec = 17
    d1 = 4.84143144246472090e+00
    d2 = -1.16032004402742839e+00
    d3 = -1.03622044471123109e-01
    print(Decimal(4.84143144246472090e+00))
    print(d1 == Decimal(4.84143144246472090e+00))

File: pyjl/tests/tests_new-py2many/rounding_test.jl
using decimal: *
function main()
getcontext().prec = 17
d1 = 4.841431442464721
d2 = -1.1603200440274284
d3 = -0.10362204447112311
println(Decimal(4.841431442464721));
println(d1 == Decimal(4.841431442464721) ? ("True") : ("False"));
end

main()
